### ISSUE TITLE
Use ji-rcn / Jinote

### ISSUE THREAD

**Issue by davidryan59** [daveryan23/retuner_musical_instrument#1]
New npm package `ji-rcn` has been developed to take care of calculating note names.

Want to use this for the note labels, at the moment the note label calculations are duplicated in ReTuner.

**Comment by davidryan59** [comment]
1) Had a go at installing ji-rcn in ReTuner, and using it to generate button labels. However, having some trouble... Installing JI (and getting rid of previous notation stuff) means that now the absolute fraction and the absolute notation are wrong.

Solution might be to enhance ji-rcn in order to:
- Get parseNotation on the class API
- (This can be used to parse just the SPACEBAR key here)
- Then get a method to setPositionFromParseNotation or something. Pass in a Peo (from parseNotation), this is equivalent to supplying a start notation, and then the rest calculates.

2) Also going to have to think more about tuning. We've got a slider to retune base position of SPACEBAR
which is equivalent to tuning. However, ji-rcn cannot edit the tuning! This is likely to have to change...

In other words - ji-rcn as written doesn't serve the application it was written for!
Redo some stuff on ji-rcn, then come back here and finish off ReTuner efficiently.
